lib/archethic/shared_secrets.ex

```diff
@@ -116,4 +116,47 @@ defmodule Archethic.SharedSecrets do
         :biometric
     end
   end
+
+  @ss_gen_key :node_shared_secrets_gen_addr
+  @spec persist_gen_addr(:node_shared_secrets | :origin) :: :ok
+  def persist_gen_addr(:node_shared_secrets) do
+    addr =
+      Archethic.Crypto.node_shared_secrets_public_key(0)
```

Comment:
This will crash as the daily nonce is not loaded.
You should as mentioned earlier to leverage post self-repair operation to look at the first genesis address of the type shared secrets from the DB.